### PR TITLE
Survival box MRE flask + MRE flask nerf

### PR DESCRIPTION
**Pull request by K-Dynamic**
## About the PR

- Replaces plastic bottled water in Survival Boxes with MRE flask
- MRE flask volume reduced from 50u to 30u

## Why / Balance

Explosions spill water bottles inside bags and cause slipping when running away. Replacing the plastic bottle with MRE flasks prevents this and protects drinks after said explosions.

MRE flasks volume reduced to match that of other drinks.

## Technical details
yml changes only
- `Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml` (replaced plastic water bottles with MRE flasks in survival boxes)
- `Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_flasks.yml` (changed description and reduced volume)

## Media

mre flask

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Survival boxes now contain MRE flasks instead of plastic water bottles
- tweak: MRE flask volume reduced from 50u to 30u

### CODE REVIEW

**Comment by ps3moira** [comment]
Something iconic about the plastic water bottle, flask feels to military

**Comment by Aisu9** [comment]
why reducing the holding capacity tho, be it 50 or 30u it cannot cause such a difference or tactical advantage right?

**Comment by Aisu9** [comment]
> > why reducing the holding capacity tho, be it 50 or 30u it cannot cause such a difference or tactical advantage right?
> 
> I would've kept it but the inconsistency kinda bugged me, plus it would make people less likely to leave their dept and get drinks from vending machines or the bar

I know drawing comparison with SS13 is kinda bad, but every bottles on 13 hold 50units, the MRE in this setting could be filled with only 30u and the maximum capacity left untouched don't you think?

**Comment by thebadman4662** [comment]
> > I know drawing comparison with SS13 is kinda bad, but every bottles on 13 hold 50units, the MRE in this setting could be filled with only 30u and the maximum capacity left untouched don't you think?
> 
> I think almost all bottles in SS14 have 30u capacity, the MRE flask had its own yml entry here making it 50u.
> 
> I'll probably ask maintainers to see about bumping volumes for all bottles including the MRE flask, no guarantees though

1x1 bottles are 30u, yes... 1x2 are usually 150-250 with exception of water bottle that is 30.

**Comment by Golinth** [comment]
>1x2 are usually 150-250

Beakers are 50u, the 2x2 large beaker is only 200u, what 1x2 item is 250u??

**Comment by thebadman4662** [comment]
> > 1x2 are usually 150-250
> 
> Beakers are 50u, the 2x2 large beaker is only 100u, what 1x2 item is 250u??

ice buckets ate 100u, some plastic drinks bottles are 150 for sure as they were part of my nukie build(why nukies have no bottles in nukie world 😭) but I believe there were some bigger ones somewhere else?

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_flasks.yml [hunk 1]
```diff
@@ -23,17 +23,11 @@
   parent: FlaskBase
   id: DrinkMREFlask
   name: MRE flask
-  description: An old military flask, filled with the finest contents for soldiers.
+  description: A surplus military flask, tapped off with water of unknown origins.
```
Comment (UbaserB) [inline Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_flasks.yml:26]: tapped off? filled sounds better imo

**Review by UbaserB — APPROVED** [review]

**Comment by slarticodefast** [comment]
Why is the flask in the survival kit empty?
And this almost completely removes the normal plastic bottles from the game, the only way to get them are now the bar supply crate and some rare random spawns.

**Comment by thebadman4662** [comment]
> Are plastic bottles a bounty?

They are source of plastic and there isn't much plastic/glass you can get otherwise without relying on station or vandalizing public and departament areas.

**Comment by UbaserB** [comment]
> Why is the flask in the survival kit empty? 

you’re so much better at spotting mistakes than me lol, i need to up my game

**Comment by SaphireLattice** [comment]
Personally I don't think I've ever thrown the bottle away. It's really useful to have around. Any explosion though...

I think most of the package wrappers should give plastic honestly. What the hell do they give right now?

**Comment by superjj18** [comment]
> > I know drawing comparison with SS13 is kinda bad, but every bottles on 13 hold 50units, the MRE in this setting could be filled with only 30u and the maximum capacity left untouched don't you think?
> 
> 
> 
> I think almost all bottles in SS14 have 30u capacity, the MRE flask had its own yml entry here making it 50u.
> 
> 
> 
> I'll probably ask maintainers to see about bumping volumes for all bottles including the MRE flask, no guarantees though

Drink shaker bartenders use is the same size but holds 100u

**Comment by thebadman4662** [comment]
> Personally I don't think I've ever thrown the bottle away. It's really useful to have around. Any explosion though...
> 
> I think most of the package wrappers should give plastic honestly. What the hell do they give right now?

Most trash is made out of steel, no matter how ridiculous it is in the context, glass/plastic trash is the exception someone needs to manually set pretty much.

**Comment by slarticodefast** [comment]
I guess I can bring this up for a maintainer vote to see which one we like more.

**Comment by slarticodefast** [comment]
Hey, I brought this up for a maintainer vote and the result was 6-1 in favor of closing this. The main reason was that the water bottle is more fitting thematically than the military themed MRE flask. Thank you for your contribution.

**Comment by slarticodefast** [comment]
I don't really get why it should be blast resistant in the first place. Yes, it breaks when you get hit by an explosion, but bags don't have explosion restistance for a reason. You shouldn't just be immune to explosions and be able to walk away unscathed.

**Pull request closed without merging**